io/disk/rawread.py

```diff
@@ -38,6 +38,11 @@ def setUp(self):
         checking install of required packages and extract and
         compile of rawread suit.
         """
+        self.disk = self.params.get("disk", default=None)
+
+        if not self.disk:
+            self.cancel("Missing disk %s in OS" % self.disk)
```

Comment:
@AYUSHJAIN951 I think it is not issing in the OS rather user did not provied the disk.
So please change the message to like "User needs to provide the disk", or "disk not provided" or "please provide the disk to run the test". any similar message looks good.

Please change it accordingly and re-send the Patch.